grid/recruitment/get-stage-changes/profile.supr

```diff
@@ -0,0 +1,198 @@
+"""
+Retrieve Stage Changes
+Lists stage changes for a candidate.
+"""
+name = "recruitment/get-stage-changes"
+version = "1.0.0"
+
+"""
+Get Stage Changes
+Retrieves a list of stages for specific candidate throughout his recruitment process. The candidate does not have to be applied for a job.
+"""
+usecase GetStageChanges {
+  input {
+    """
+    Candidate Id
+    Unique identifier of a candidate.
+    """
+    candidateId! string!
```

Comment:
Can I use lead ID as well?

Reply:
This might be problematic with different providers. Depends on whether `Lead` and `Candidate` are different entities with separate id pools.

If they are not the same entity, it does not make sense to put lead ID here as `Lead` will always be just in `sourced` stage, but if they share the ids, it's possible and we could rename this field.

Reply:
Our goal is to harmonise the APIs so that the client using OneSDK doesn't have to care about the implementation detail on the provider side.

50inTech is very specific about this use case:
> Fetching the status changes of a previously sent candidate (applicant or sourced)

We should accept both Candidate ID and Lead ID.

In case that Lead and Candidate are different entities with separate ID pools on the provider side. Can we add ID of the pool to Candidate or Lead entity ID when retrieving it using `ListCandidates`, `CreateCandidate or `CreateLead` use cases?

Reply:
Good point. I'm not sure if I understand the question, do you mean to alter the id returned from those use cases and parse it in this use case to differ between candidate/lead?

Reply:
Proposed solution: Use cases `ListCandidates`, `CreateCandidate` or `CreateLead` return compound ID for example: "CANDIDATE:1" or "LEAD:10". Use case `GetStageChanges` parses the ID and handles returns result differently for lead IDs and candidate IDs.

Reply:
👍🏻

Reply:
Updated documentation, didn't find better naming then `candidateId` 🤷🏻‍♂️